### DISCUSSION TITLE
Input to show face

### DISCUSSION THREAD

**Discussion by vishalnp37** [neperfepx/neper#720] [Q&A] [answered]
I have a simple tessellation, I am operating on the areas of the faces of the tessellations , I want to be able to visualize the the face that i am working on. 
I have used the following command 
neper -V n100.tess -showface "faceareas<0.005" -showedge "cell_shown||domtype==1" -print img5

if i use just -showface "areas<0.005" , all the cells are visible? 

![image](https://github.com/neperfepx/neper/assets/128611901/78ef30d8-5659-4935-9d6c-dc608e55e976)

**Comment by rquey** [comment, marked as answer]
A face has only one area, so `-showface "area<0.005"`.
